src/pages/AddItemPage/AddItemPage.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Container,
+  FlexContainer,
+  SectionTitle,
+} from "../../styles/CommonStyles";
+import styled from "styled-components";
+import InputItem from "../../components/UI/InputItem";
+import TagInput from "../../components/UI/TagInput";
+import ImageUpload from "../../components/UI/ImageUpload";
+
+const TitleSection = styled(FlexContainer)`
+  margin-bottom: 16px;
+`;
+
+const InputSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+
+  @media ${({ theme }) => theme.mediaQuery.tablet} {
+    gap: 24px;
+  }
+`;
+
+function AddItemPage() {
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
+  const [price, setPrice] = useState("");
```

Comment:
price 의 경우 숫자라고 생각이 드는데 초기값이 "" 라서 타입병기해주시면 좋을 거 같아요!
```suggestion
  const [price, setPrice] = useState<string>("");
```